src/sbvr-api/sbvr-utils.ts

```diff
@@ -1020,6 +1031,31 @@ export class PinejsClient extends PinejsClientCore {
 	}) {
 		return (await runURI(method, url, body, tx, req, custom)) as object;
 	}
+
+	public post<TResource extends keyof M & string>(
+		params: {
+			resource: TResource;
+			options?: Params<M[TResource]>['options'] & { returnResource?: true };
+		} & Params<M[TResource]>,
+	): Promise<M[TResource]['Read']>;
+	public post<TResource extends keyof M & string>(
+		params: {
+			resource: TResource;
+			options: Params<M[TResource]>['options'] & { returnResource: boolean };
+		} & Params<M[TResource]>,
+	): Promise<Pick<M[TResource]['Read'], 'id'>>; // TODO: This should use the primary key rather than hardcoding `id`
+	/**
+	 * @deprecated POSTing via `url` is deprecated
+	 */
```

Comment:
Do we have an alternative/proposal/idea for doing `canAccess` now that posting via url is deprecated?

Reply:
Not yet, but this is to highlight any cases that come up so we can work on either converting them to standard ways that we've added support for or if they don't already have a way then to make sure we create one. However it would still be possible to change from
```ts
api.post({
	url,
	...
})
```
to
```
api.request({
	method: 'POST',
	url,
	...
})